backend/endpoints/rooms.py

```diff
@@ -62,3 +64,89 @@ async def create_room(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail="unable to create room",
     )
+
+
+def remove_mem(room_obj: dict, mem_id: str, org_id: str):
```

Comment:
This seem like a util function, just that you won't have these things that make it more of an endpoint e.g raise HTTPException, return JSON response, so that the util function can be more generic in nature and easily adaptable to anywhere it will be used.